Release Notes/Release Notes v2.11.0-beta1.md

```diff
@@ -0,0 +1,58 @@
+# .NET Driver Version 2.11.0-beta1 Release Notes
+
+This is a beta release for the 2.11.0 version of the driver.
+
+The main new features in 2.11.0-beta1 support new features in MongoDB 4.4. These features include:
+
+* Support for all new
+  [``$meta``](https://docs.mongodb.com/manual/reference/operator/projection/meta/)
+  projections: `randVal`, `searchScore`, `searchHighlights`,
+  `geoNearDistance`, `geoNearPoint`, `recordId`, `indexKey` and
+  `sortKey`
+* Support for passing a hint to update commands as well as
+  `findAndModify` update and replace operations
+* Support for `allowDiskUse` on find operations
+* Support for `MONGODB-AWS` authentication using Amazon Web Services
+  (AWS) Identity and Access Management (IAM) credentials
+* Support for stapled OCSP (Online Certificate Status Protocol) (macOS only)
+
+Other new additions and updates in this beta include:
+
```

Comment:
@rstam Should we include a bullet about https://jira.mongodb.org/browse/CSHARP-2724? If so, would you be able to provide a summary bullet in noun-clause form? 😺

Reply:
Oh man... we should get the docs for that in beta 1.

Maybe if I stop doing code reviews I can get at least a draft of that in today.

Reply:
* New GuidRepresentationMode setting to opt-in to the new V3 GuidRepresentation mode

But maybe put this at the end of the list, not first.

Reply:
Tweaked mildly, appended to the end of the list. @rstam, please resolve this thread if you feel it looks good!